travis/large-requirements.txt

```diff
@@ -12,7 +12,7 @@ onnxruntime==0.3.0;
 mleap==0.8.1
 mxnet==1.5.0
 pandas<=0.23.4
-pyarrow==0.12.1
+pyarrow==0.17.0
```

Comment:
Don't we want something like >= 0.17.0 in order to detect potential future regression ?